torch/quantization/fx/quantize.py

```diff
@@ -81,15 +83,19 @@
 def insert_observer(
         node: Node, observer: torch.quantization.ObserverBase,
         model: torch.nn.Module,
-        activation_post_process_map: Dict[str, torch.quantization.ObserverBase],
+        activation_post_process_map: Dict[str, List[torch.quantization.ObserverBase]],
+        activation_post_process_indexes: Dict[str, int],
         env: Dict[Any, Any], observed_graph: Graph, load_arg: Callable,
-        observed_node_names_set: Set[str]):
+        observed_node_names_set: Set[str],
+        quants: Dict[str, List[Tuple[DefaultQuantizeHandler, Callable]]]):
     """Insert observer for node by modifying the observed_graph and
        attach observer module to the model
        Args:
          node: Node
          observer: observer/fake_quantize module instance
     """
+    if not model.training and isinstance(observer, torch.quantization.FixedQParamsFakeQuantize):
+        return
```

Comment:
just curious, what's the context on this one?  Maybe we can add a comment?

Reply:
Sure I can add a comment. reason is fixedqparams fake quant only needs to be there for training, we can either do this check here or in _find_quant.